### ISSUE TITLE
Add setting to avoid saving images to storage when downloading

### ISSUE THREAD

**Issue by Eddyall-1** [Bionus/imgbrd-grabber#2683]
**Is your feature request related to a problem? Please describe**

I primarily use Grabber to send picture to a Szurubooru instance with tags already filled in. Pictures aren't meant to be kept on the computer Grabber is running on and these will simply pile on and fill space on the PC instead of the server the Szurubooru is running on.

**Describe the solution you'd like**

Add an option to have Grabber skip the step where it saves images to disk and instead just fill in it's own database that the image was "downloaded" while still triggering any script the user has like the one that sends the image to a Szurubooru server.

**Describe alternatives you've considered**

Emptying the folder Grabber downloads to periodically manually or with a script, however the goal is to avoid writing the images to disk/SSD in the first place, to save on time + wear and tear on the PC's storage.

**Comment by Bionus** [comment]
> Emptying the folder Grabber downloads to periodically manually or with a script, however the goal is to avoid writing the images to disk/SSD in the first place, to save on time + wear and tear on the PC's storage.

I see what you mean, but the files are passed to the `szurubooru.js` script via a temporary file in any case. So unless passing it as  some kind of string-converted blob, or convert this script to C++, you'll always get files written to the disk.

Another alternative you didn't mention is deleting the file directly from the `szurubooru.js` file BTW (wouldn't avoid wear, but you won't need some kind of cron cleaning, and it'll handle errors well since it won't delete files if an error occurred uploading to szurubooru).

**Comment by Eddyall-1** [comment]
Right, if the scripts gets the file from storage and not memory/a link already, that wouldn't work without modding the default script. Good point.
